tests/test_manifest_validation.py

```diff
@@ -174,3 +175,41 @@ def test_validate_manifest_block(manifest_validation_config):
     block_requirement = {"versions": {"guppy": "quay.io/cdis/guppy:0.3.0"}}
     ok = validate_manifest_block(block_requirement, manifest_validation_config["block"])
     assert not ok, "guppy without guppy block should pass validation"
+
+
+def test_global_requirements_validation(manifest_validation_config):
```

Comment:
missing test case: "hatchery" not in "versions" + "netpolicy" not in "global" should pass validation